conf/conf.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package conf
+
+import (
+	"embed"
+)
+
+//go:embed app.ini **/*
+var EmbedFS embed.FS
```

Comment:
```suggestion
var Files embed.FS
```

I feel like `EmbedFS` isn't a great name, `conf.Files` looks much nicer, what do you think?